Mathlib/Algebra/Parity.lean

```diff
@@ -134,6 +134,7 @@ theorem Even.neg_pow : Even n → ∀ a : α, (-a) ^ n = a ^ n := by
   simp_rw [← two_mul, pow_mul, neg_sq]
 #align even.neg_pow Even.neg_pow
 
+@[simp]
```

Comment:
Why not add `@[simp]` to `Even.neg_pow` and `Odd.neg_pow` instead?

Reply:
Better, thanks!